Packs/Active_Directory_Query/ReleaseNotes/1_3_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Fixed an issue in ***iam-update-user*** command where the command failed whenever the *allow-enable* argument was set to true.
```

Comment:
```suggestion
- Fixed an issue in the ***iam-update-user*** command where the command would fail when the *allow-enable* argument was set to true.
```